h2o-assemblies/main/build.gradle

```diff
@@ -88,6 +88,11 @@ dependencies {
         api('org.apache.commons:commons-configuration2:2.10.1') {
             because 'Fixes CVE-2024-29131'
         }
+        api('dnsjava:dnsjava:3.6.0') {
```

Comment:
@valenad1

This fix is for CVE-2024-25638.  Can you add that as

because 'Fixes CVE-2024-25638'

Reply:
Done